apps/fxc-front/public/static/archives.html

```diff
@@ -12,7 +12,7 @@
   <meta charset="utf-8" />
```

Comment:
Plese delete this file. It's not used any more.
I left it around to use when porting to the new arch but should have deleted it as part of https://github.com/vicb/flyxc/commit/4a1aef2f3acf8dc5984b64885df34ac8ffdd2562